rust/core-lib/Cargo.toml

```diff
@@ -17,6 +17,8 @@ toml = "0.4"
 notify = { optional = true, version = "4.0" }
 regex = "1.0"
 memchr = "2.0.1"
+strum = "0.13.0"
+strum_macros = "0.13.0"
```

Comment:
This allows to print an enum as string.

Reply:
Would it not be feasible to, instead of adding another dependency, to print enums as strings in long form? I.e. [the answers to this SO question](https://stackoverflow.com/questions/28024373/is-there-a-way-to-print-enum-values)

Reply:
This works for enums that don't have any fields. But if they have fields then their values would be part of the string:

```rust
#[derive(Debug)]
pub enum E {
    Delete,  // -> this works fine
    Edit(usize, usize) //  -> would print for example: "Edit(3, 4)"
}
```

Reply:
That's a fair point. I hadn't thought of that.